pennylane/devices/default_gaussian.py

```diff
@@ -598,15 +598,15 @@ def poly_quad_expectations(mu, cov, wires, device_wires, params, hbar=2.0):
     groenewald_correction = np.sum([np.linalg.det(hbar * A[:, m][n]) for m in modes for n in modes])
     var -= groenewald_correction
 
-    return ex, var
+    return var, ex
```

Comment:
Similar question as for the mean photon case regarding the change of order here.